examples/fft_vs_geom_movie.py

```diff
@@ -0,0 +1,348 @@
+                    if j == 0:
+                        continue
+                    ztext[j-1].set_text("Z{:d} = {:5.3f}".format(j+1, ab))
+
+                # Calculate simple estimate of ellipticity
+                mom_fft = simple_moments(fft_img)
+                mom_phot = simple_moments(geom_img)
+                e_fft = ellip(mom_fft)
+                e_phot = ellip(mom_phot)
+
+                Is = ("$I_x$={:6.3f}, $I_y$={:6.3f}, $I_{{xx}}$={:6.3f},"
+                      " $I_{{yy}}$={:6.3f}, $I_{{xy}}$={:6.3f}")
+                I_fft.set_text(Is.format(mom_fft['Ix'], mom_fft['Iy'],
+                                         mom_fft['Ixx'], mom_fft['Iyy'], mom_fft['Ixy']))
+                I_phot.set_text(Is.format(mom_phot['Ix'], mom_phot['Iy'],
+                                          mom_phot['Ixx'], mom_phot['Iyy'], mom_phot['Ixy']))
+                etext_fft.set_text("$e_1$={:6.3f}, $e_2$={:6.3f}, $r^2$={:6.3f}".format(
+                                   e_fft['e1'], e_fft['e2'], e_fft['rsqr']))
+                etext_phot.set_text("$e_1$={:6.3f}, $e_2$={:6.3f}, $r^2$={:6.3f}".format(
+                                    e_phot['e1'], e_phot['e2'], e_phot['rsqr']))
+
+
+                fft_mom[i] = (mom_fft['Ix'], mom_fft['Iy'],
+                              mom_fft['Ixx'], mom_fft['Iyy'], mom_fft['Ixy'],
+                              e_fft['e1'], e_fft['e2'], e_fft['rsqr'])
+
+                geom_mom[i] = (mom_phot['Ix'], mom_phot['Iy'],
+                              mom_phot['Ixx'], mom_phot['Iyy'], mom_phot['Ixy'],
+                              e_phot['e1'], e_phot['e2'], e_phot['rsqr'])
+
+                writer.grab_frame(facecolor=fig.get_facecolor())
+
+                bar.update()
+
+    def symmetrize_axis(ax):
+        xlim = ax.get_xlim()
+        ylim = ax.get_ylim()
+        lim = min(xlim[0], ylim[0]), max(xlim[1], ylim[1])
+        ax.set_xlim(lim)
+        ax.set_ylim(lim)
+        ax.plot(lim, lim)
+
+    # Centroid plot
+    fig, axes = plt.subplots(1, 2, figsize=(10, 6))
+    axes[0].scatter(fft_mom[:, 0], geom_mom[:, 0])
+    axes[1].scatter(fft_mom[:, 1], geom_mom[:, 1])
+    axes[0].set_title("Ix")
+    axes[1].set_title("Iy")
+    for ax in axes:
+        ax.set_xlabel("Fourier Optics")
+        ax.set_ylabel("Geometric Optics")
+        symmetrize_axis(ax)
+    fig.tight_layout()
+    fig.savefig(args.out+"centroid.png", dpi=300)
+
+    # Second moment plot
+    fig, axes = plt.subplots(1, 3, figsize=(16, 6))
+    axes[0].scatter(fft_mom[:, 2], geom_mom[:, 2])
+    axes[1].scatter(fft_mom[:, 3], geom_mom[:, 3])
+    axes[2].scatter(fft_mom[:, 4], geom_mom[:, 4])
+    axes[0].set_title("Ixx")
+    axes[1].set_title("Iyy")
+    axes[2].set_title("Ixy")
+    for ax in axes:
+        ax.set_xlabel("Fourier Optics")
+        ax.set_ylabel("Geometric Optics")
+        symmetrize_axis(ax)
+    fig.tight_layout()
+    fig.savefig(args.out+"2ndMoment.png", dpi=300)
```

Comment:
So I've thought about these plots a bit more, and I think they're consistent with missing an additional convolution by an isotropic kernel, which is what the second kick will be once we implement it.  I.e., the first moments are all spot-on, as are the Ixy moments, and the Ixx and Iyy moments are too small (by a fixed amount) without the second kick.

As for the last set of plots, I think the problem here is just that the image size is too small (0.6 arcsec by default).  If you run `python fft_vs_geom_movie.py --diam 8.36 --obscuration 0.61 --nlayer 6 --sigma 0.05 --size 3` instead, then you get the following figure:

![fft_vs_geom_2ndmoment](https://cloud.githubusercontent.com/assets/3650485/21727967/b42887a0-d3f8-11e6-94bf-a3f70f6e0c04.png)

The other piece these comparisons are missing is that they're all instantaneous PSFs, so I'll try running a few time-integrated simulations to see what happens there.

I've also realized that this approximation is probably really only appropriate for atmospheric phase screen PSFs, and not pure optical PSFs.  I.e., using the geometric approximation for shooting WFIRST or Euclid photons is probably not sufficiently accurate, since the high frequency modes don't get averaged down over time the way they do for an atmospheric PSF.  So I propose to make `geometric_shooting=True` the default for `PhaseScreenPSF`, which is where atmospheric PSFs originate, but use `geometric_shooting=False` for the `OpticalPSF` class.  An interesting question then is:  What is the impact of using `geometric_shooting=True` for the optical part of a combined atmosphere+optics PSF?  I'll test that out some too.